components/creator-feed/creator-feed-button.tsx

```diff
@@ -17,14 +17,33 @@ import { getAddress } from "viem";
 export default function CreatorFeedButton({
   hypercertId,
   creatorAddress,
+  disabledForChain = false,
```

Comment:
What about features that we want to disable for other reasons? Like for example disabled in production but not on testnets, etc.

Reply:
We'll add them when that's needed, for now we only need a boolean check based on the users connected chain. The return object of the flag can be more complex